documentation/framework/sitting-framework.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: wiki
+title: Sitting Framework
+description: Explains how to set-up sitting furniture with ACE3 sitting system.
+group: framework
+order: 5
+parent: wiki
+---
+
+## 1. Adding sitting support to a chair
+
+<div class="panel callout">
+    <h5>Note:</h5>
+    <p>Unfinished! What you see below will not work in ACE 3.2.1</p>
```

Comment:
... then throw this file out and put it in a different PR that will be merged after 3.3.0